lib/routemaster/redis_broker.rb

```diff
@@ -25,18 +27,42 @@ def cleanup
       _cleanup
     end
 
+    # Allow to inject pre-built Redis clients
+    #
+    # Before storing a new connection, ensure that any previously
```

Comment:
I think you meant:

> Before storing a new connection, **ensures** that any previously

I originally read it as "you need to close old connections yourself before calling this".